packages/widget-v2/src/pages/SwapPage/SwapDetailModal.tsx

```diff
@@ -0,0 +1,213 @@
+import { css, styled } from "styled-components";
+import { createModal, ModalProps } from "@/components/Modal";
+import { Column, Row } from "@/components/Layout";
+import { SmallText } from "@/components/Typography";
+import { RouteArrow } from "@/icons/RouteArrow";
+import { SwapPageFooterItems } from "./SwapPageFooter";
+import { useAtom, useAtomValue } from "jotai";
+import { skipChainsAtom, skipRouteAtom } from "@/state/skipClient";
+import { useMemo } from "react";
+import { swapSettingsAtom } from "@/state/swapPage";
+import { formatUSD } from "@/utils/intl";
+import { SLIPPAGE_OPTIONS } from "@/constants/widget";
+import { getClientOperations, OperationType } from "@/utils/clientType";
+
+export const SwapDetailModal = createModal((modalProps: ModalProps) => {
+  const { data: route } = useAtomValue(skipRouteAtom);
+  const { data: chains } = useAtomValue(skipChainsAtom);
+  const [swapSettings, setSwapSettings] = useAtom(swapSettingsAtom);
+  const chainsRoute = useMemo(() => {
+    return route?.chainIDs.map((chainID) => chains?.find((chain) => chain.chainID === chainID));
+  }, [route, chains]);
+
+  const clientOperations = route && getClientOperations(route.operations);
+
+
+  const axelarTransferOperation = useMemo(() => {
+    if (!clientOperations) return;
+    return clientOperations?.find((item) => item.type === OperationType.axelarTransfer);
+  }, [clientOperations]);
+
+  const hyperlaneTransferOperation = useMemo(() => {
+    if (!clientOperations) return;
+    return clientOperations?.find((item) => item.type === OperationType.hyperlaneTransfer);
+  }, [clientOperations]);
+
+  const axelarFee = useMemo(() => {
+    if (axelarTransferOperation) {
+      const { feeAmount, feeAsset, usdFeeAmount } =
+        axelarTransferOperation;
+      if (!feeAmount || !feeAsset) return;
+      const computed = (
+        +feeAmount / Math.pow(10, feeAsset.decimals || 18)
+      ).toLocaleString("en-US", {
+        maximumFractionDigits: 6,
+      });
```

Comment:
we can use `convertTokenAmountToHumanReadableAmount(tokenAmount, decimals);` for this right?